src/main/java/com/xceptance/neodymium/util/LighthouseUtils.java

```diff
@@ -0,0 +1,217 @@
+package com.xceptance.neodymium.util;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WindowType;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.PathNotFoundException;
+
+import io.qameta.allure.Allure;
+
+/**
+ * Powered by <a href="https://developer.chrome.com/docs/lighthouse/overview?hl=de">Lighthouse</a> (Copyright Google)
+ */
+public class LighthouseUtils
+{
+    /**
+     * Creates a <a href="https://developer.chrome.com/docs/lighthouse/overview?hl=de">Lighthouse</a> report
+     * (Copyright Google) of the current URL and adds it to the Allure report.
+     * 
+     * @param driver
+     *            The current webdriver
+     * @param reportName
+     *            The name of the Lighthouse report attachment in the Allure report
+     * @throws Exception
+     */
+    public static void createLightHouseReport(WebDriver driver, String reportName) throws Exception 
+    {
+        // validate that lighthouse is installed
+        try 
+        {
+            if (System.getProperty("os.name").toLowerCase().contains("win")) 
+            {
+                ProcessBuilder builder = new ProcessBuilder();
+                builder = new ProcessBuilder("cmd.exe", "/c", Neodymium.configuration().lighthouseBinaryPath(), "--version");
+                Process p = builder.start();
+                int errorCode = p.waitFor();
+                
+                if (errorCode != 0) 
+                {
+                    throw new IOException();
+                }
+            }
+            else if (System.getProperty("os.name").toLowerCase().contains("linux") || System.getProperty("os.name").toLowerCase().contains("mac"))
+            {
+                new ProcessBuilder(Neodymium.configuration().lighthouseBinaryPath(), "--version").start();
```

Comment:
for linux please change this to call the shell by using `new ProcessBuilder("sh", "-c", Neodymium.config....`